modules/edu.gemini.seqexec.model/src/main/scala/edu/gemini/seqexec/model/Model.scala

```diff
@@ -139,7 +139,7 @@ object Model {
     * Represents a queue with different levels of details. E.g. it could be a list of Ids
     * Or a list of fully hydrated SequenceViews
     */
-  case class SequencesQueue[T](queue: List[T])
+  case class SequencesQueue[T](conditions: Conditions, queue: List[T])
```

Comment:
Ok, now I see why this broke on the client. `SequencesQueue` is/was a simple marker for a list of somehting. This change is fine though. Did it break the client?